mailet/api/src/main/java/org/apache/mailet/DsnParameters.java

```diff
@@ -53,7 +53,8 @@ public class DsnParameters {
     public static final String ORCPT_PARAMETER = "ORCPT";
     public static final String ENVID_PARAMETER = "ENVID";
     public static final String RET_PARAMETER = "RET";
-
+    public static final String HOLDFOR_PARAMETER = "HOLDFOR";
+    public static final String HOLDUNITL_PARAMETER = "HOLDUNITL";
```

Comment:
Future release parameters are not DSN parameters. Please move them to a dedicated file.

Reply:
I actually see the same constants declared in `FUTURERELEASEParameters` and I think it's enough (no need to have them duplicated)

Just remove those from `DsnParameters` class :)

Reply:
Oh I see, so I created a FututeReleaseParameters class and move HOLDFOR_PARAMETER and HOLDUNTIL_PARAMETER to this class in my local repo, you will see it in next commit.